### ISSUE TITLE
Slack notifications have no icon

### ISSUE THREAD

**Issue by danirabbit** [elementary/panel-notifications#206]
## What Happened

Notifications sent from the Slack desktop app in Flatpak don't have an icon

## Expected Behavior

Even if Slack is doing something dumb, we should at least have a fallback icon.

I haven't confirmed if this happens in the server too, but it likely does so we'll want to make sure to port the fix

## Steps to Reproduce

1. Install slack desktop from Flatpak
2. Receive a notification
3. Notice that it has no icon in the notification center

**Comment by andirsun** [comment]
Same behavior with Telegram Flatpak version

**Comment by danirabbit** [comment]
Yeah Telegram must be using legacy notifications and not the portal. It fails to identify itself properly. But we do get avatar images and the fallback app icon, so I think this is as fixed as it can be
